### ISSUE TITLE
Document localized descriptions

### ISSUE THREAD

**Issue by yolile** [open-contracting/standard_extension_template#25]
Can we add to the readme a section to mention how the description and title fields can be localized?
Like: 
```javascript
"description" : 
[
    "es": "Descripción"
    "en": "Description"
]
```
Just for mention the format and correct form to do that and mention that this can be done.

**Comment by jpmckinney** [comment]
@yolile Yes, please make a PR.

**Comment by yolile** [comment]
Done in #26

**Comment by jpmckinney** [comment]
Closed by #26
